src/overlays/actors/ovl_En_Sw/z_en_sw.h

```diff
@@ -14,39 +14,51 @@ typedef struct EnSw {
     /* 0x0190 */ EnSwActionFunc actionFunc;
     /* 0x0194 */ ColliderJntSph collider;
     /* 0x01B4 */ ColliderJntSphElement colliderElements[1];
-    /* 0x01F4 */ Color_RGBA8 unk_1F4;
+    /* 0x01F4 */ Color_RGBA8 limb4Color; // never set past black.
     /* 0x01F8 */ Vec3s jointTable[30];
     /* 0x02AC */ Vec3s morphTable[30];
-    /* 0x0360 */ u8 unk_360;
-    /* 0x0364 */ Vec3f unk_364;
-    /* 0x0370 */ Vec3f unk_370;
-    /* 0x037C */ Vec3f unk_37C;
-    /* 0x0388 */ s16 unk_388;
-    /* 0x038A */ s16 unk_38A;
-    /* 0x038C */ s16 unk_38C;
-    /* 0x038E */ s16 unk_38E;
-    /* 0x0390 */ s16 unk_390;
-    /* 0x0392 */ s16 unk_392;
-    /* 0x0394 */ s16 unk_394;
-    /* 0x0396 */ char unk_396[0x42];
-    /* 0x03D8 */ MtxF unk_3D8;
+    /* 0x0360 */ u8 goldInAir; // set when revealed, unset when landing.
+    /* 0x0364 */ Vec3f surfaceNormal; // normal of wall/floor/ceiling it's on.
+    /* 0x0370 */ Vec3f unk_370; // used for rotate calculation
+    /* 0x037C */ Vec3f unk_37C; // used for rotate calculation
+    /* 0x0388 */ s16 animTimer;
+    /* 0x038A */ s16 animVar; // used to determine crawl and "death bounce" animations.
+    /* 0x038C */ s16 waitTimer;
+    /* 0x038E */ s16 crawlTimer;
+    /* 0x0390 */ s16 attackTimer;
+    /* 0x0392 */ s16 painTimer;
+    /* 0x0394 */ s16 deathFlames;
```

Comment:
```suggestion
    /* 0x0394 */ s16 deathFlamesTimer;
```
or `deathFlamesCount`